tests/integration/011_policy/expected_outputs.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do we require this file to exist? 🤔

Reply:
It certainly appears so.

https://github.com/env0/terraform-provider-env0/blob/main/tests/harness.go#L115